README.md

```diff
@@ -0,0 +1,188 @@
+# Enumerable
+## Getting Started
+This repository includes files ruby code on Enumerable and 3 related methods any , filter all:
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+- [📖 \[Enumerable\] ](#-enumerable-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Enumerable <a name="about-project"></a>
+
+**Enumerable** In this project, we utilize the Ruby language to write a class List and module enumerable.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+<summary>RUBY</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">Ruby Terminal</a></li>
+  </ul>
+</details>
```

Comment:
- [ ] This link is misleading, I was redirecte to a wrong page. Kindly justify it. ℹ️